src/controllers/admin/email.controller.ts

```diff
@@ -30,3 +30,15 @@ export const sendEmailController = async (
     throw err
   }
 }
+
+export const enableEmailReminderHandler = async (
+  req: Request,
+  res: Response
+): Promise<void> => {
+  try {
+    res.status(200).json({ message: 'Reminder enabled' })
```

Comment:
What is the purpose of this?

Reply:
this is a API endpoint but implemented yet. I'll implement this soon as possible :)